artifactory/services/utils/fileinfoutils.go

```diff
@@ -14,8 +14,9 @@ type FileHashes struct {
 
 type FileInfo struct {
 	*FileHashes
-	LocalPath       string `json:"localPath,omitempty"`
-	ArtifactoryPath string `json:"artifactoryPath,omitempty"`
+	LocalPath               string `json:"localPath,omitempty"`
+	ArtifactoryPath         string `json:"artifactoryPath,omitempty"`
+	RelativeArtifactoryPath string `json:"relativeArtifactoryPath,omitempty"`
```

Comment:
I suggest renaming RelativeArtifactoryPath to InternalArtifactoryPath.
I think this needs to be chabnged in additional places in the code.